plugins/hls-hlint-plugin/test/Main.hs

```diff
@@ -285,3 +316,9 @@ hlintConfigWithFlags flags =
   where
     unObject (Object obj) = obj
     unObject _            = undefined
+
+knownBrokenForHlintOnGhcLib :: String -> TestTree -> TestTree
```

Comment:
It's not immediately obvious to me what these mean or why it's these versions. Haddock?

Reply:
Hmm it is a somewhat long history, will try to add a concise comment although the entire thing should be explained in the plugin README or something

Reply:
Not sure if i explained it clearly :thinking: